src/js/app.js

```diff
@@ -57,20 +57,30 @@ App = {
 
       var account = accounts[0];
 
+      // user cannot transfer tokens to himself
+      if (toAddress.toLowerCase() == account.toLowerCase()) {
+        alert('Sorry, you cannot transfer token to yourself');
```

Comment:
I would reword these two lines to:

`Sorry, you cannot transfer tokens to yourself` (make token plural)